### ISSUE TITLE
Question from a disabled gamer

### ISSUE THREAD

**Issue by Tezzle** [Ryochan7/DS4Windows#304]
Hi! I'm a disabled gamer and I've used DS4Windows back in 2015(or 2016) primarily for accessibility related reasons. I set up multiple profiles with macros that switch between them, I set up buttons on right stick up/left/right! But apparently it doesn't work anymore..I want to ask if this version works fine on Windows 10, if it's safe to use and if it has all the features I've used?

**Comment by Limyx826** [comment]
Hey this issue is the same as mine! #281
